### ISSUE TITLE
Reverse dependencies incomplete

### ISSUE THREAD

**Issue by timokau** [Mic92/nix-review#51]
When using `nix-review` 2.0 to review https://github.com/NixOS/nixpkgs/pull/52020, it only builds `simgear`. But `flightgear` is a package that depends on `simgear`, so it should be rebuilt too.

**Comment by Mic92** [comment]
The problem here is that ofborg does not list flightgear in its evaluation: https://gist.github.com/GrahamcOfBorg/e700616454179424bce9508ccc014c54

Is it possible that flightgear is a proprietary package?

**Comment by timokau** [comment]
No, gpl2. But

```nix
    hydraPlatforms = []; # disabled from hydra because it's so big
```

may be relevant.

**Comment by Mic92** [comment]
Ok. Not having the evaluate nixpkgs is a huge time-saver in nix-review. Therefore I would rather like to see ofborg also including packages with `hydraPlatforms = []` rather then having to evaluate it locally.

**Comment by timokau** [comment]
So `nix-review` somehow gets the list of packages to build from ofBorg? Doesn't it need to evaluate nixpkgs anyways to actually build the packages?

**Comment by Mic92** [comment]
No. It just evaluates the packages it builds, which is much faster then evaluating all packages twice (once on master and once on the pr branch).

**Comment by timokau** [comment]
Makes sense. @grahamc do you think it makes sense to ignore `hydraPlatforms` for ofBorg, or should we just ignore this edge case?

**Comment by grahamc** [comment]
Off the cuff I think it makes sense. It may prove difficult if this introduces new errors for some reason, but in principle yes.
